Allow attach/detach db to IModelDb connection (#7530)
iTwin/itwinjs-core: docs/changehistory/NextVersion.md

  ...
}
```

## Attach/detach db

> Allow to attach a ECDb/IModel to connection and run ECSQL that combines data from both databases.

Reviewer: ```suggestion
Allow the attachment of an ECDb/IModel to a connection and running ECSQL that combines data from both databases.
```

Example of attaching a snapshot to a master file and running a query that combines data from both databases:
```ts

>  const master = SnapshotDb.openFile(masterFile);

Reviewer: ```suggestion
     const master = SnapshotDb.openFile(masterFile);
```

Example of attaching a snapshot to a master file and running a query that combines data from both databases:
```ts

>      const master = SnapshotDb.openFile(masterFile);

Reviewer: Sorry; looks like the other lines are three-space indented, while my suggested change was 4. I think the indent for all lines should be either 4 spaces or 2, but I'm not sure which is better.

Reviewer: Never mind; looks like my suggested change somehow had 5 spaces, so just fix that.

  ...
}
```

## Attach/detach db

Allow the attachment of an ECDb/IModel to a connection and running ECSQL that combines data from both databases.

Example of attaching a snapshot to a master file and running a query that combines data from both databases:

> ```ts

Reviewer 2: Please consider moving this example code to example-code-snippets and linking to it from here and more importantly from the `attachDb` method's API docs.

Author: I have added snippet and link it to NextVersion.md & attach/detach function api doc.
